cs2pr

```diff
@@ -87,6 +91,10 @@ foreach ($root as $file) {
         $line = (string) $error['line'];
         $message = (string) $error['message'];
 
+        if($showSource) {
+            $message = ((string) $error['source']).': '.$message;
```

Comment:
This requires a array_key_exists() or similar check beforehand, as not every error will have a „source“?